system/lib/libc/musl/src/math/ceil.c

```diff
@@ -1,11 +1,13 @@
 #include "libm.h"
 
+#ifndef __wasm__
```

Comment:
Why the new ifdefs?

Reply:
The stuff inside these block is not used at all in the `__wasm__` case below